spec.bs

```diff
@@ -1493,6 +1509,21 @@ give to the [=Relying Party=] (e.g., the credit card number).
 NOTE: See [issue #77](https://github.com/w3c/secure-payment-confirmation/issues/77)
       for discussion on possible spec changes related to this section.
 
+## User opt out ## {#sctn-user-opt-out}
+
+The API provides an option {{SecurePaymentConfirmationRequest/showOptOut}} for
+the developer to indicate to the user agent that that the user should be
+provided with a button to opt out of the process. When the button is selected,
+and an {{OptOutError}} is returned to the caller to indicate the user's intent
+to opt out, it is then up to the caller to act on the opt out, e.g. by clearing
+payment information stored for the user.
+
+NOTE: This is not intended to be a mechanism to delete browser data or
```

Comment:
I think this is actually key text for this section, and so shouldn't be in a `NOTE`!

Reply:
Done